tests/qos/test_tunnel_qos_remap_nvidia.py

```diff
@@ -0,0 +1,231 @@
+    dscp_to_queue_mapping = [
+        # DSCP, Outer DSCP, Expected queue
+        (8, 8, 0),
+        (0, 0, 1),
+        (33, 33, 1),
+        (3, 2, 2),
+        (4, 6, 6),
+        (46, 46, 5),
+        (48, 48, 7)
+    ]
+    with allure.step("Prepare test parameters and switch one server on the lower tor to active"):
+        dualtor_meta = dualtor_info(ptfhost, rand_selected_dut, rand_unselected_dut, tbinfo)
+        lower_tor_mg_facts = rand_unselected_dut.get_extended_minigraph_facts(tbinfo)
+        upper_tor_mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
+        vlan_interface = list(lower_tor_mg_facts['minigraph_vlans'].keys())[0]
+        # use the last interface as the active interface on lower tor
+        lower_tor_active_interface = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members'][-1]
+        lower_tor_active_server_ptf_port = lower_tor_mg_facts['minigraph_ptf_indices'][lower_tor_active_interface]
+        logger.info("Toggle the interface {} to active on the lower tor.".format(lower_tor_active_interface))
+        toggle_ports(rand_unselected_dut, [lower_tor_active_interface], "active")
+        toggle_ports(rand_selected_dut, [lower_tor_active_interface], "standby")
+        lower_tor_uplink_interfaces = {}
+        t1_ptf_ports = []
+        for portchannel in lower_tor_mg_facts['minigraph_portchannels'].values():
+            for member in portchannel['members']:
+                ptf_indice = lower_tor_mg_facts['minigraph_ptf_indices'][member]
+                t1_ptf_ports.append(ptf_indice)
+                lower_tor_uplink_interfaces[ptf_indice] = member
+        # randomly choose a standby server on lower tor
+        lower_tor_standby_interface = random.choice(
+            lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members'][:-1])
+        lower_tor_standby_server = lower_tor_mg_facts['minigraph_neighbors'][lower_tor_standby_interface]['name']
+        lower_tor_standby_server_ip = lower_tor_mg_facts['minigraph_devices'][lower_tor_standby_server]['lo_addr']
+
+        rand_selected_dut_name = upper_tor_mg_facts['inventory_hostname']
+        rand_unselected_dut_name = lower_tor_mg_facts['inventory_hostname']
+        upper_tor_loopback0_ip = upper_tor_mg_facts['minigraph_devices'][rand_selected_dut_name]['lo_addr']
+        lower_tor_loopback0_ip = lower_tor_mg_facts['minigraph_devices'][rand_unselected_dut_name]['lo_addr']
+
+    with allure.step("Start the traffic test"):
+        for dscp, outer_dscp, queue in dscp_to_queue_mapping:
+            pkt = simple_tcp_packet(
+                eth_dst=dualtor_meta['vlan_mac'],
+                ip_src='1.1.1.1',
+                ip_dst=lower_tor_standby_server_ip,
+                ip_dscp=dscp,
+                ip_ecn=1,
+                ip_ttl=64
+            )
+            # The ttl of inner_frame is decreased by 1
+            pkt.ttl -= 1
+            ipinip_packet = simple_ipv4ip_packet(
```

Comment:
Confused by the intention of this test case. What is it you want to validate here? If you want to validate the packet encapsulation, you can send it directly to standby. If you want to validate the remap, why are you crafting the encap packets yourself?